.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules/
 .DS_Store
 .idea
 *.swp
+.vscode
```

Comment:
Please remove this. I like it but would rather it be part of another pull request.

Reply:
Ok